ios/brave-ios/Sources/Brave/Frontend/Settings/Tabs/PrivateTabsView.swift

```diff
@@ -77,13 +77,7 @@ struct PrivateTabsView: View {
               if newValue {
                 tabManager?.saveAllTabs()
               } else {
-                if let tabs = tabManager?.allTabs.filter({ $0.isPrivate }) {
-                  SessionTab.deleteAll(tabIds: tabs.map({ $0.id }))
-                }
-
-                if tabManager?.privateBrowsingManager.isPrivateBrowsing == true {
-                  tabManager?.willSwitchTabMode(leavingPBM: true)
-                }
+                tabManager?.removeAllForCurrentMode(isActiveTabIncluded: true)
```

Comment:
Does it need to delete all the SessionTab?

Reply:
i kept the same as the old logic, it deletes all SessionTab that are private https://github.com/brave/brave-core/blob/3064726d7d8dc6e9dd4e0ccfcdb176b1861d0e6d/ios/brave-ios/Sources/Brave/Frontend/Settings/Tabs/PrivateTabsView.swift#L81